components/x-eventpromo/src/ImagesContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import { h, Component } from '@financial-times/x-engine';
+import Image from './Image';
+import styles from './ImagesContainer.scss';
+
+class ImagesContainer extends Component {
+
+	constructor(props, ...args) {
+		super(props, ...args);
```

Comment:
What is the purpose of the rest arg here? Can there be anything other than `props`?

Reply:
react also passes `context` through here